packages/amplify-category-auth/src/provider-utils/supported-services.ts

```diff
@@ -1141,7 +1141,7 @@ export const supportedServices = {
       {
         key: 'loginwithamazonAppIdUserPool',
```

Comment:
for what its worth it took me a while to figure out that this one was the correct login with amazon prompt text, whereas the "amazonAppId" version was legacy. codebase could be commented to reflect this better.